### DISCUSSION TITLE
Specify flags globally

### DISCUSSION THREAD

**Discussion by m-kuhn** [microsoft/vcpkg#24053] [Q&A] [answered]
I'd like to enable address sanitizing globally for all ports.

I tried doing so by modifying the triplet and adding

```
set(ENV{LDFLAGS} "-fsanitize=address")
set(ENV{CXXFLAGS} "-fsanitize=address -fno-omit-frame-pointer")
set(ENV{CFLAGS} "-fsanitize=address -fno-omit-frame-pointer")
```

However, libspatialite chokes on this because it overwrites `LDFLAGS` in its portfile.

Is there a better (compatible, recommended) way to do this?
Is this an issue with libspatialite that should be solved?

**Comment by autoantwort** [comment, marked as answer]
I think it is better to set the following: https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md#vcpkg_cxx_flags

**Reply by m-kuhn** [reply]
Much better indeed. Thanks!
